src/lib/CashlinkStore.ts

```diff
@@ -30,7 +30,11 @@ export class CashlinkStore extends Store<Cashlink, CashlinkEntry> {
     }
 
     protected toEntry(cashlink: Cashlink): CashlinkEntry {
-        return cashlink.toObject();
+        const entry = cashlink.toObject();
+        // delete fee and currently unused contactName which are unnecessary to store
+        delete entry.fee;
+        delete entry.contactName;
+        return entry;
```

Comment:
Is a cashlink's `.toObject()` used anywhere currently where the fee and contactName is used? If not, I suggest to remove the fee and contactName from the return value of `toObject` be done with it. Would also avoid the abomination of a function name below (`expectEqualIgnoringFeeAndContactName`).

Reply:
`toObject` should return an object representation of the cashlink from which it can be recreated.
To omit data there just to avoid it being included in an independent class would be a hack.
The data is used to recreate cashlinks on reload in the ledger flow and to restore the inputs in `CashlinkCreate`.

Reply:
In my opinion the cashlink's `.toObject()` should omit the fee (and maybe contactName, as long as it's unused) in the first place. The fee is so unimportant to the cashlink itself that it does not need to be exported.

Reply:
The fee was added to `toObject` when it became necessary to have it there.
It is used for recreating a Cashlink from the history store to the static store on reload in `CashlinkCreate` and `SignTransactionLedger`.
I don't see why the cashlink lib should not export these values which are a part of describing a cashlink instance and are needed to recreate an equivalent instance.
To make things slightly nicer in the `CashlinkStore` is not really a valid reason.
A compromise would be to add a flag to `toObject` that omits the `fee` and `contactName`.